test/common/http/header_map_impl_test.cc

```diff
@@ -424,9 +424,8 @@ TEST_P(HeaderMapImplTest, AllInlineHeaders) {
     INLINE_REQ_RESP_STRING_HEADERS(TEST_INLINE_STRING_HEADER_FUNCS)
   }
   {
-    // No request trailer O(1) headers.
-  }
-  {
+      // No request trailer O(1) headers.
+  } {
```

Comment:
This seems unrelated.  Can we revert this change to eliminate unnecessary diffs?  Or git checkout  the main version and call it a day.

Reply:
courtesy of clang formatter